process/track/baseBlockTrack.go

```diff
@@ -680,3 +684,25 @@ func (bbt *baseBlockTrack) initNotarizedHeaders(startHeaders map[uint32]data.Hea
 
 	return nil
 }
+
+func (bbt *baseBlockTrack) doWhitelistIfNeeded(metablock *block.MetaBlock) {
+	shardID := bbt.shardCoordinator.SelfId()
+	if shardID == core.MetachainShardId {
```

Comment:
we are not doing whitelisting for miniblocks on the metachain? Meta can process miniblocks also.

Reply:
Actually metachain does not have a superior chain which could tell him which miniblocks are withelist and which are not. This will notify only shards nodes with the new whitelisted miniblocks which are inside the new shard headers notarized by meta.

Reply:
Yeah, so we can spam metachain's miniblock topic freely (within the antiflood limits)

Reply:
if we were to whitelist metachain miniblocks as well, it would be too late as these have already been agreed upon in consensus.